src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20161026_superowner_workspace_acls.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="migrate-superowners-1">
+        <sql stripComments="true">
+            create table super_owners_temp_no_realm (workspace_id BINARY(16), project_name varchar(254), group_email varchar(254), group_name varchar(254))
+        </sql>
+    </changeSet>
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="migrate-superowners-2">
+        <sql stripComments="true">
+            insert into super_owners_temp_no_realm select ws.id, ws.namespace,
```

Comment:
i don't think going to drop these tables _yet_ because they're very useful for the sync-group migration script.

i'll add a note in the "drop PROJECT_USERS table" jira ticket to also drop these tables.